packages/web-react/src/types/link.ts

```diff
@@ -9,6 +9,8 @@ import {
 
 export type LinkTarget = '_blank' | '_self' | '_parent' | '_top';
 
+export type UnderlinedOptions = 'always' | 'never' | 'hover';
```

Comment:
Can you convert it also to constant and use it in `useLinkStyleProps`? Please, look at the `dictionaries` and use it the same way :-)

Reply:
converted to constants